### PR TITLE
Event card

### PR DESCRIPTION
**Pull request by Nim000**

### CODE REVIEW

**Comment by Axetyy** [comment]
The Image doesn't fit well, also why delete everything in About Us?

File: src/app/about/page.tsx [hunk 1]
```diff
@@ -1,26 +1,48 @@
-import React from "react";
-import { Button, Container, Stack, Typography } from "@mui/material";
-import { Roboto } from "next/font/google";
-import { Box } from "@mui/material";
-import { GenericSection } from "@/components/organism/GenericSection/GenericSection";
+import React from 'react';
+import { Button, Container, Stack, Typography } from '@mui/material';
```
Comment (baragan30) [inline src/app/about/page.tsx:2]: We don't need so many imports. Delete them

File: src/app/about/page.tsx [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
-const VISION_IMAGE : string = `/images/about/Our-Vision-Image.jpg`
-const VISION_TITLE_TEXT : string = `Viziunea Noastră`
+      trăind valorile pe care le promovăm și implicându-ne activ în comunitate.`,
+  `Ne angajăm să fim un factor de schimbare pozitivă în educație. Alătură-te nouă în această călătorie și să fim împreună forța 
+      care schimbă viitorul educației în România!`,
+];
+const VISION_IMAGE: string = `/images/about/Our-Vision-Image.jpg`;
+const VISION_TITLE_TEXT: string = `Viziunea Noastră`;
 const HomePage: React.FC = () => {
   return (
```
Comment (baragan30) [inline src/app/about/page.tsx:30]: Don't all comment all the existing features . Try to integrate yours in what exists. I think the place of the Event Card is in the our events page not in about us page. What do you think? Please create that page!

**Comment by baragan30** [comment]
Please try to merge this branch to development branch not to main branch

**Review by baragan30 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**